### ISSUE TITLE
Slider. Proper mouse handling

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-jb#325]
```
import androidx.compose.desktop.Window
import androidx.compose.material.Slider
import androidx.compose.runtime.getValue
import androidx.compose.runtime.mutableStateOf
import androidx.compose.runtime.remember
import androidx.compose.runtime.setValue

fun main() = Window {
    var value by remember { mutableStateOf(5f) }
    Slider(
        value = value,
        valueRange = 0f..10f,
        onValueChange = { value = it }
    )
}
```

When we drag the slider with mouse, dragging don't start immedeately:
https://user-images.githubusercontent.com/5963351/106154944-327ae880-6191-11eb-9bc6-b3581d0e04c8.mp4

We should introduce proper mouse handling to Slider component. Because non-immediate start of dragging should only happened with touch gestures.

See also https://github.com/JetBrains/compose-jb/issues/320#issuecomment-769166209

**Comment by igordmn** [comment]
Should be fixed in 1.0.0-beta5

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6248">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
